sentry-ruby/lib/sentry/net/http.rb

```diff
@@ -30,13 +30,13 @@ def request(req, body = nil, &block)
         return super if from_sentry_sdk?
 
         Sentry.with_child_span(op: OP_NAME, start_timestamp: Sentry.utc_now.to_f) do |sentry_span|
-          set_sentry_trace_header(req, sentry_span)
+          request_info = extract_request_info(req)
+          set_sentry_trace_header(req, sentry_span, request_info)
```

Comment:
I feel we can perform the `propagate_trace?` check here without changing `set_sentry_trace_header`.
And we can use `Sentry.configuration` too.

Reply:
this'll change a bit again as part of https://github.com/getsentry/sentry-ruby/issues/2056 so I'll clean the whole logic there if it's fine?

Reply:
Will it be implemented soon? I think it'll be fine if we ship them in the sam release.

Reply:
yea like this/next week